Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,34 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            BMI();
+            BMI();
+        }
+
+        private static void BMI()
```

Comment:
Function name should be a verb. CalculateBmi is a better name.
Also, use PascalCase (even for abreviations, first letter should be capitalized and then for every next word it should also be capitalized)